whylog/converters/exceptions.py

```diff
@@ -0,0 +1,9 @@
+from whylog.exceptions import WhylogError
+
+
+class UnsupportedConverter(WhylogError):
+    def __init__(self, converter_class_name):
+        self.converter_class_name = converter_class_name
+
+    def __str__(self):
+        return 'This whylog version do not handle %s. Please upgrade Whylog' % self.converter_class_name
```

Comment:
It depends on what you mean by "upgrade". If you mean changing whylog code by hand, it is ok. If you mean downloading the newest version of whylog from the internet, you're wrong.